recovery/translation_nl.ts

```diff
@@ -362,22 +362,22 @@ Hierbij worden alle bestaande bestanden verwijderd.</translation>
     <message>
         <location filename="mainwindow.cpp" line="117"/>
         <source>Waiting for SD card (settings partition)</source>
-        <translation type="unfinished"></translation>
+        <translation>Wachten op SD kaart (settings partitie)</translation>
     </message>
     <message>
         <location filename="mainwindow.cpp" line="130"/>
         <source>Mounting settings partition</source>
-        <translation type="unfinished"></translation>
+        <translation>Mounten van de settings partitie</translation>
```

Comment:
Elders gebruik je koppelen i.p.v. mounten.
Beide is mogelijk, maar beter is om consequent 1 van de 2 te kiezen.